04_testing/severity_analysis.R

```diff
@@ -964,3 +965,396 @@ write_csv(
     "2022_hno_compr_adjustment_pop.csv"
   )
 )
+
+###################################################
+#### RELATIONSHIP SECTORAL SUM & INTERSECTORAL ####
+###################################################
+
+df_sum_sev <- df %>%
```

Comment:
not all areas in a country have the same sectors, some areas miss a sector or two. therefore, the sum of sectors might show very low for areas that miss 2 sectors and have all the other sectors as 4 or 5

Reply:
you could make it as % of sum severity over sum of all sectors if they had 5, something like that to account for those areas that miss a sector or two, meanwhile check MOZAMBIQUE some areas have 3 sectors while others have up to 8

Reply:
Hmm, should filter out first and then average?

Reply:
I believe average makes more sense as it accounts of the number of sectors available in each area, not sure if that's enough or if we need more aggregation method to show more versions of the distribution